packages/panels/resources/views/components/logo.blade.php

```diff
@@ -1,19 +1,38 @@
 @php
     $brandName = filament()->getBrandName();
-    $brandLogo = filament()->getBrandLogo()
+    $brandLogo = filament()->getBrandLogo();
+    $brandLogoHeight = filament()->getBrandLogoHeight() ?? '1.25rem';
```

Comment:
IMO this should be `1.5` or `1.75`

Reply:
Yeah I wasn't too sure about 1.25 vs 1.5. However, 1.25 looked slightly better with the Filament logo.

Reply:
I think we should lean towards 1.5, so I am going to swap it